src/com/puppetlabs/puppetdb/scf/storage.clj

```diff
@@ -1094,48 +1094,70 @@
     (dissoc row-map :environment_id)
     row-map))
 
+(defn normalize-resource-event
```

Comment:
To me, 'normalize' means that there are many different shapes of input that we're converting to be one shape of output... but I don't think that's what's happening here.